src/BookAnalizer/BookStatisticCalculator.java

```diff
@@ -0,0 +1,78 @@
+package BookAnalizer;
+
+import java.io.*;
+import java.util.*;
+
+public class BookStatisticCalculator {
+    public void analyzeBook(File file) {
+        try {
+            Set<String> uniqueWords = readWordsFromFile(file);
+            List<String> filteredWords = filterWords(uniqueWords);
+            Map<String, Integer> wordCount = countWordOccurrences(filteredWords);
```

Comment:
means only unique words with length > 2 are counted?

Reply:
Yep, filter for length done before this method. Changed the variable name a bit